src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Field.cs

```diff
@@ -51,56 +50,48 @@ public override BoundNode VisitFieldAccess(BoundFieldAccess node)
         /// produces fieldAccess(field=Item1, receiver=fieldAccess(field=Rest, receiver=ValueTuple for tuple))
         /// </summary>
         private BoundExpression MakeTupleFieldAccess(
-            CSharpSyntaxNode syntax, 
-            TupleFieldSymbol tupleField, 
+            CSharpSyntaxNode syntax,
+            FieldSymbol tupleField, 
             BoundExpression rewrittenReceiver,
             ConstantValue constantValueOpt,
             LookupResultKind resultKind,
             TypeSymbol type)
         {
-            var tupleType = (TupleTypeSymbol)tupleField.ContainingSymbol;
+            var tupleType = tupleField.ContainingType;
 
-            int fieldRemainder;
-            int loop = TupleTypeSymbol.NumberOfValueTuples(tupleField.Position, out fieldRemainder);
+            NamedTypeSymbol currentLinkType = tupleType.TupleUnderlyingType;
+            FieldSymbol underlyingField = tupleField.TupleUnderlyingField;
 
-            NamedTypeSymbol currentLinkType = tupleType.UnderlyingTupleType;
+            if ((object)underlyingField == null)
+            {
+                // Use-site error must have been repoted elsewhere.
```

Comment:
Thanks, I'll fix in the next change.

---
In reply to: [61790864](https://github.com/dotnet/roslyn/pull/11006#discussion_r61790864) [](ancestors = 61790864)